framework/commercetools/lib/normalize.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  CommercetoolsProducts,
+  CommerceToolsProductPrice,
+  Product,
+  ProductVariant,
+  CommercetoolsProductVariant,
+  ProductPrice,
+  Images,
+  ProductImage,
+} from '../types/product'
+import type {
+  Cart,
+  CommercetoolsCart,
+  CommercetoolsLineItems,
+  LineItem,
+} from '../types/cart'
+
+function setProductPrice(prices: CommerceToolsProductPrice): ProductPrice {
+  return {
+    value: prices.value.centAmount,
+    currencyCode: prices.value.currencyCode,
+    retailPrice: 0,
+    salePrice: 0,
+    listPrice: 0,
+    extendedListPrice: 0,
+    extendedSalePrice: 0,
+  }
+}
+
+function setImage(images: Images[]): ProductImage[] {
+  return images.map((image) => {
+    return { url: image.url }
+  })
+}
+
+function setVariants(
+  variants: CommercetoolsProductVariant[],
+  published: boolean
+): ProductVariant[] {
+  return variants.map((variant) => {
+    return {
+      id: variant.id,
+      options: [],
+      availableForSale: published,
+    }
+  })
+}
+
+export function normalizeProduct(data: CommercetoolsProducts): Product {
+  const price = setProductPrice(data.masterVariant.prices[0])
```

Comment:
You can get the `data.masterVariant.price` value instead. If it does not exist, then you can fallback to the first price you find.